ansible_collections/arista/avd/molecule/cv_deploy/workspace_force_true.yml

```
@@ -0,0 +1,61 @@
+---
+- name: Converge - cv_deploy with cv_submit_workspace_force = true
+  hosts: SITE1_FABRIC
```

Add group INACTIVE to the scope (host avd-ci-core1 is dynamically added to it below)
```suggestion
  hosts: SITE1_FABRIC:INACTIVE
```